modules/combined/test/tests/reference_residual/gold/reference_residual_perfgraph_out.csv

```diff
@@ -0,0 +1,4 @@
+time,elapsed,res_calls
```

Comment:
We can't have a test checked in with specific wall times or even number of residual calls. This will definitely be brittle. Perhaps you can just check that the file exists, or if you want to look at the content you can check in a "custom comparison" file that will be really permissive on the values (maybe a range of 0-20 seconds or between 30-50 residual calls).

Reply:
Thanks for the review @permcody --I've got a relative tolerance of 1000 in the tests file now (which might be the same as the custom comparison but less ideal). I'll take a look at how to check for the presence of a file and redo the commit

Reply:
Sorry I missed that... Relative tolerance of 1000 is fine. That might be better than "existence" because it'll check that the columns don't change. Either solution is fine - I'll leave that up to you.

Reply:
In that case I'm going to go with the lower entropy solution and leave it as is - letting laziness run out